scripts/ctype_translator.py

```diff
@@ -14,8 +18,89 @@
     c_uint32: 'integer',
     c_uint64: 'integer',
     c_float: 'single',
-    c_double: 'double'
+    c_double: 'double',
+    c_void_p: 'raw'
+})
+
+
```

Comment:
```suggestion
# comprehensive list of R types can be found here: https://cran.r-project.org/doc/manuals/r-release/R-ints.html
# in short:
# - NILSXP : NULL
# - CHARSXP : internal character strings
# - LGLSXP : logical vectors
# - INTSXP : integer vectors
# - REALSXP : numeric vectors
# - RAWSXP : raw vector
```

Provide a documentation in comment about the datatypes in R

Reply:
Added with extra explaination on NILSXP and RAWSXP